netdata.spec.in

```diff
@@ -40,7 +40,7 @@ AutoReqProv: yes
 #
 # Conditional build:
 %bcond_without  systemd  # systemd
-%bcond_with     netns    # build with netns support (cgroup-network)
+%bcond_without  netns    # build with netns support (cgroup-network)
```

Comment:
@Ferroin I am not sure why we need this condition at all 🤷‍♂️ Do you remember/know?

Reply:
It’s a leftover from CentOS 6, but it’s also probably not worth making the effort right now to completely remove it.

Reply:
Correction, this is planned for removal as part of https://github.com/netdata/netdata/pull/13927, once I finally get back to working on that. So no need to drop it here and make this PR more complicated than it needs to be.